tests/scene/test_text_edit.h

```diff
@@ -3086,7 +3086,7 @@ TEST_CASE("[SceneTree][TextEdit] context menu") {
 	text_edit->get_viewport()->set_embedding_subwindows(true); // Bypass display server for drop handling.
 
 	text_edit->set_size(Size2(800, 200));
-	text_edit->set_line(0, "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec varius mattis leo, sed porta ex lacinia bibendum. Nunc bibendum pellentesque.");
+	text_edit->set_line(0, "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec godous mattis leo, sed porta ex lacinia bibendum. Nunc bibendum pellentesque.");
```

Comment:
Didn't want to have to figure out how to update the test if the string length changed, and didn't want to keep `varius` as an exception, so picked a random 6 letter latin-like word. "godotus" was 7 :(